virt_lightning/__init__.py

```diff
@@ -188,6 +194,37 @@ def init_network(self):
         elif self.conn.getURI().startswith("qemu:///system"):
             print("system")
 
+    def init_storage_pool(self, storage_pool):
+        try:
+            self.storage_pool_obj = self.conn.storagePoolLookupByName(storage_pool)
+            if not self.get_storage_dir().is_dir():
+                raise Exception("Missing storage directory:", self.get_storage_dir())
+            return
+        except libvirt.libvirtError as e:
+            if e.get_error_code() == libvirt.VIR_ERR_NO_STORAGE_POOL:
+                print("Storage pool is missing.")
+                pass
+
+        if os.geteuid() == 0:
+            storage_dir = pathlib.PosixPath("/var/lib/virt-lightning/pool")
```

Comment:
storage_dir must be configurable.

Reply:
Agreed, but this can also be done later. This patch is big enough.